plugins/faustwp/tests/integration/replacement/test-replacement-functions.php

```diff
@@ -19,4 +36,24 @@ public function test_domain_replacement_enabled_returns_true_when_filtered_to_be
 		$this->assertTrue( domain_replacement_enabled() );
 		remove_filter( 'faustwp_domain_replacement_enabled', '__return_true' );
 	}
+
+	public function test_normalize_sitemap_url() {
```

Comment:
Our other test function names tend to follow a pattern that fully spells out what exactly is being tested. In this case, it might be something like `test_normalize_sitemap_url_replaces_frontend_uri_with_home_url()`. It's long and weird, but I think it helps communicate the intent and the expectations of a specific test. It would also provide clarity if we were to add cases to test different branches of `normalize_sitemap_url()` (a test called `test_normalize_sitemap_url_does_not_replace_when_frontend_uri_not_set()`, for example). There's no hard rule on it and wouldn't consider it blocking, but just something to consider 🙂

Reply:
Related: if you run phpunit with the `--testdox` option, it will print a readable version of the test name. Example of the current name:
```
WPE\FaustWP\Tests\Replacement\ReplacementFunctionsTestCases
 ✔ Domain replacement enabled returns false
 ✔ Domain replacement enabled returns true when filtered to be true
 ✔ Normalize sitemap url
 ✔ Normalize sitemap entry
 ```
 
 Named like Andrew suggested (third item):
 ```
 WPE\FaustWP\Tests\Replacement\ReplacementFunctionsTestCases
 ✔ Domain replacement enabled returns false
 ✔ Domain replacement enabled returns true when filtered to be true
 ✔ Normalize sitemap url does not replace when frontend uri not set
 ✔ Normalize sitemap entry
 ```

Reply:
Dry vs Damp in unit tests is what we should follow here. I will make a note for the upcoming PR Workshop to discuss this in detail:
https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
@apmatthews I am all for having tests be more readable; Updated!
@mindctrl Thanks for the tip! I've updated our composer script to include that flag.
@theodesp Thank you!